.eslintrc

```diff
@@ -1,10 +1,15 @@
 {
     "parser": "babel-eslint",
     "extends": ["standard", "standard-react"],
+    "globals": {
+        "__DEV__": false
```

Comment:
let's just have __DEV__ be a global so we don't need that comment to ignore it all the time